extensions/Worker.Extensions.Storage.Queues/src/StorageQueuesExtensionStartup.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Core;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+[assembly: WorkerExtensionStartup(typeof(StorageQueuesExtensionStartup))]
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public class StorageQueuesExtensionStartup : WorkerExtensionStartup
+    {
+        public override void Configure(IFunctionsWorkerApplicationBuilder applicationBuilder)
+        {
+            if (applicationBuilder == null)
+            {
+                throw new ArgumentNullException(nameof(applicationBuilder));
+            }
+
+            applicationBuilder.Services.Configure<WorkerOptions>((workerOption) =>
+            {
+                workerOption.InputConverters.RegisterAt<QueueMessageConverter>(0);
```

Comment:
Can you remind me why we need to register this explicitly if we already have `InputConverterAttribute` on `QueryTriggerAttribute`?

Reply:
We do not need to register, I think this is just left over from the previous implementation - will clean this up!